client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -143,7 +151,7 @@ const MarketplaceThankYou = ( { productSlug }: { productSlug: string } ) => {
 		}
 
 		// Update the menu after the plugin has been installed, since that might change some menu items.
-		if ( areAllPluginsOnSite ) {
+		if ( isAllDataFetched ) {
```

Comment:
Nitpick: Should we use `products` as terminology everywhere as above instead of using `data`?

Reply:
Done https://github.com/Automattic/wp-calypso/pull/74125/commits/9af806d1cc524adee6f3fed98f9e0af52d2b105b